app/controllers/hyrax/downloads_controller.rb

```diff
@@ -29,7 +29,12 @@ def show
     # OVERRIDE Hyrax 2.9.6 for IRUS Analytics
     def item_identifier_for_irus_analytics
       # return the OAI identifier
-      "#{CatalogController.blacklight_config.oai[:provider][:record_prefix].call(self)}:#{params[:id]}"
+      # We must send the parent work id to IRUS otherwise it gets confused
+      # This works but hits fcrepo twice :/
+      parent_work_id = ActiveFedora::Base.find(params[:id]).parent.id
```

Comment:
Using `ActiveFedora::Base.find` will pull from Fedora.  We may want to consider `ActiveFedora::Base.where(id: params[:id]).first`.

Further, the parent might be `nil` which would raise an exception.  We've found that the parent may not be set (see https://github.com/scientist-softserv/iiif_print/blob/d8246e1d95f340ae769dbd05be5a500930c9162a/app/services/iiif_print/pluggable_derivative_service.rb#L115-L119)

Reply:
Aha... `where` rather than `find` I knew there must be a better way. Thanks @jeremyf !